terraform/README.md

```
@@ -0,0 +1,126 @@
+# Terraform 운영 가이드
+
+본 디렉터리의 인프라 코드를 만지기 전에 한 번 읽어주세요.
+
+## State 의 역할
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)